gettsim/interface.py

```diff
@@ -545,11 +717,15 @@ def _partial_parameters_to_functions(functions, params):
             for i in arguments
             if i.endswith("_params") and i[:-7] in params
         }
+        if partial_params:
+            partial_func = functools.partial(function, **partial_params)
 
-        partialed_functions[name] = (
-            functools.partial(function, **partial_params)
-            if partial_params
-            else function
-        )
+            # Make sure that rounding parameter attribute is kept
```

Comment:
Changed the comment to:

```
            # Make sure that rounding parameter attribute is transferred to partial function.
            # Otherwise, this information would get lost.
            
            ```
            
            Clearer?